pages/standards-guidelines/wcag/non-web-ict.md

```diff
@@ -44,24 +44,64 @@ Quick link:
 
 ## Introduction {#intro}
 
-<cite>Guidance on Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT)</cite> describes how the Web Content Accessibility Guidelines (WCAG) can be applied to non-web information and communications technologies (ICT), including documents and software. WCAG2ICT is a W3C Group Note that provides informative guidance (not normative requirements).
+<cite>Guidance on Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT)</cite> describes how the Web Content Accessibility Guidelines (WCAG) can be applied to non-web information and communications technologies (ICT), including documents and software. WCAG2ICT is the acronym used for the W3C Guidance on Applying WCAG 2 to Non-web Information and Communication Technology Group Note.
+
+The WCAG2ICT Group Note provides:
+* [Informative](https://www.w3.org/TR/WCAG22/#dfn-informative) **guidance** that is not [normative](https://www.w3.org/TR/WCAG22/#dfn-normative) and that does not set requirements.
```

Comment:
Not sure if linking to WCAG's definition of normative and informative is a good idea. We should think about how much we want to link to other places. This is a potential distraction for readers. Aren't normative and informative self-explanatory in this context?

```suggestion
* **Informative guidance** that is not normative and that does not set requirements.
```

Reply:
I think this was something that the link was in the WCAG2ICT Note and when I copied in the text, and the link remained. However, there are those of us that work in the standards world and we know what normative vs. informative is. Not sure policy makers and others who are new to reading standards know the distinction. I don't think having a link is harmful here, and would prefer it be kept..

Reply:
Task Force discussed, we want to keep these links. Closed without action.

Reply:
@ChrisLoiselle 

Thanks. I understand TF rationale. I'd like to leave this open in case this serves as guidance for other WAI Staff that will review this.